### PR TITLE
Add debug function to ensure institution n logo images are alive

### PR DESCRIPTION
**Pull request by DannyOppenheimer**
Notes:
This is PR is only regarding issue #60 so:
- It doesn't check whether the function "debugInstitutionLogos()" works. (I'm pretty sure this function is never even used)
- It doesn't use "debugPortraits()." (Looks unfinished and unused as well)

### CODE REVIEW

**Comment by DannyOppenheimer** [comment]
Close #60

File: js/scripts.js [hunk 1]
```diff
@@ -409,6 +411,17 @@ function debugInstitutionLogos() {
     }
 }
 
+function debugInstitutionLogosImageExistence() {
+    // Check to ensure the URL to each institution's logo image hasn't moved, died, etc.
+    for(let image of logos) {
```
Comment (ErikBoesen) [inline js/scripts.js]: `for (`

File: js/scripts.js [hunk 2]
```diff
@@ -409,6 +411,17 @@ function debugInstitutionLogos() {
...
 
+function debugInstitutionLogosImageExistence() {
+    // Check to ensure the URL to each institution's logo image hasn't moved, died, etc.
+    for (let image of logos) {
+        imageExists(image[1], exists => {
+            if (!exists) {
+                console.log(`Link for the logo of ${image[0]} does not return a valid image. URL: ${image[1]}`)
+            }
+        });
+    }
```
Comment (AndrewLester) [inline js/scripts.js]: `image[0]` and `image[1]` are a little ambiguous and require someone to go find out what those properties are.
```suggestion
    for (const [ destination, url ] of logos) {
        imageExists(url, exists => {
            if (!exists) {
                console.log(`Link for the logo of ${destination} does not return a valid image. URL: ${url}`)
            }
        });
    }
```

File: js/scripts.js [hunk 3]
```diff
@@ -409,6 +411,17 @@ function debugInstitutionLogos() {
     }
 }
 
+function debugInstitutionLogosImageExistence() {
+    // Check to ensure the URL to each institution's logo image hasn't moved, died, etc.
+    for (const [ instiutionName, url] of logos) {
+        imageExists(url, exists => {
+            if (!exists) {
+                console.log(`Link for the logo of ${instiutionName} does not return a valid image. URL: ${url}`);
```
Comment (AndrewLester) [inline js/scripts.js]:
```suggestion
    for (const [ institutionName, url] of logos) {
        imageExists(url, exists => {
            if (!exists) {
                console.log(`Link for the logo of ${institutionName} does not return a valid image. URL: ${url}`);
```

**Review by AndrewLester — DISMISSED** [review]

**Review by AndrewLester — APPROVED** [review]

**Pull request merged**